src/main/java/com/wss/remediation/Utils.java

```diff
@@ -0,0 +1,15 @@
+package com.wss.remediation;
+
+import org.owasp.esapi.ESAPI;
+import org.owasp.esapi.codecs.AbstractCharacterCodec;
+import org.owasp.esapi.codecs.UnixCodec;
+
+public class Utils {
+
+
+    static String EsapiEncoder(AbstractCharacterCodec codec, String param) {
```

Comment:
please start a method name with a small letter

Reply:
Change to lower letter